tests/django_pg_migration_tools/test_timeouts.py

```diff
@@ -2,11 +2,49 @@
 from unittest import mock
 
 import pytest
-from django.db import utils
+from django.db import connection, connections, transaction, utils
+from django.test import utils as test_utils
 
 from django_pg_migration_tools import timeouts
 
 
+class _Leaky:
```

Comment:
might as well do

```
@contextlib.contextmanager
def _leaky():
    with transaction.atomic("default"):
        with connection.cursor() as cursor:
          # This will raise an exception that aborts the transaction
          # because the IntModel model has `int_field` as required field
          # and we aren't giving it.
          cursor.execute("INSERT INTO example_app_intmodel (id) VALUES (%s)", [1])